src/passes/pass.cpp

```diff
@@ -204,6 +204,9 @@ void PassRegistry::registerPasses() {
                createGUFAOptimizingPass);
   registerPass(
     "optimize-j2cl", "optimizes J2CL specific constructs.", createJ2CLOptsPass);
+  registerPass("merge-j2cl-itables",
+               "Merges itable structures into vtables to use less memory",
```

Comment:
Maybe just say "to make types more compact" which could imply both memory and size?